lib/spack/spack/cmd/uninstall.py

```diff
@@ -106,33 +106,41 @@ def find_matching_specs(env, specs, allow_multiple_matches=False, force=False):
     return specs_from_cli
 
 
-def installed_dependents(specs):
+def installed_dependents(specs, env):
     """Map each spec to a list of its installed dependents.
 
     Args:
         specs (list): list of Specs
+        env (Environment): the active environment, or None
 
     Returns:
-        (dict): mapping from spec to lists of Environments
+        (tuple): mapping from spec to sets of dependent specs
```

Comment:
This returns two things now, so the return doc should be updated

Reply:
done.